javascript/javascript-quiz.md

```diff
@@ -121,7 +121,7 @@ for (var i = 1; i <= 4; i++) {
 }
 ```
 
-- [x] .
+- [ ] .
```

Comment:
Previous answer was correct you can check code results in browser console. There is required to find code which will log sequence of numbers from 1 to 4.

Answer selected before:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/33329898/196477029-432abdab-534b-49c2-89a2-12117aedfc5b.png">

Your answer:
<img width="225" alt="image" src="https://user-images.githubusercontent.com/33329898/196477297-0b182584-ffa0-497c-80a9-22665192c1e3.png">

Reply:
you right, thanks for check
i missed checkbox location, so checked checkbox stay BEFORE its answer, i missed this